src/example-browser.html

```diff
@@ -1,6 +1,6 @@
 <script type="module">
-  import { render, html, svg } from 'uhtml/async';
-  import jspm from 'https://api.github.com/orgs/jspm' assert { type: "json" };
+  import { render, html, svg } from "uhtml/async";
+  // import jspm from 'https://api.github.com/orgs/jspm' assert { type: "json" };
```

Comment:
why are these commented?

Reply:
In dev mode. Every time we refresh, it's making a call to github. After certain calls, github started to block my requests 😬. So, I thought we could make it static and forgot to change it back